lib/jinterface/java_src/com/ericsson/otp/erlang/OtpEpmd.java

```diff
@@ -242,7 +241,7 @@ private static int r4_lookupPort(final AbstractNode node)
                 System.out.println("<- (no response)");
             }
             throw new IOException("Nameserver not responding on " + node.host()
-                    + " when looking up " + node.alive());
+                    + " when looking up " + node.alive(), e);
```

Comment:
This constructor was introduced in Java 1.6. 

I will take up a discussion about the minimal Java version to be supported by jinterface.